ndbench-core/src/main/java/com/netflix/ndbench/core/NdBenchDriver.java

```diff
@@ -219,7 +231,14 @@ public Void call() throws Exception {
                         if ((operation.isReadType() && readsStarted.get()) ||
                                 (operation.isWriteType() && writesStarted.get())) {
                             if (rateLimiter.get().tryAcquire()) {
-                                operation.process(ndBenchMonitor, keyGenerator.getNextKey());
+                                // TODO - remove this
```

Comment:
the statement is helpful for debugging.. but since it would be invoked in a tight loop and potentially skew metrics for very low latency data stores, i think we agreed to comment it out.. but leave it in the src as a hint for how to get more debugging insight..     this is what i will do on next push (unless u prefer other course of action).